cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ResponseUtils.java

```diff
@@ -57,11 +57,24 @@ static void writeResponseToOutputStream(HttpServletResponse response,
                                           boolean json,
                                           String responseMessage,
                                           KafkaCruiseControlConfig config)
+    throws IOException {
+    writeResponseToOutputStream(response, responseCode, json, null, responseMessage, config);
+  }
+
+  static void writeResponseToOutputStream(HttpServletResponse response,
```

Comment:
we should also check whether to populate schema information in header when returning [in-progress](https://github.com/linkedin/cruise-control/blob/d3619c650aaff1f3750a7d7b0f3ec5ab5f1c4741/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ProgressResult.java#L64)/[error](https://github.com/linkedin/cruise-control/blob/213fd8e348e62a7cadbfbc6b4dbe48e6700d9d12/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ResponseUtils.java#L96) response

Reply:
I moved the schema logic to `ResponseUtils` so the logic can be shared for these responses.